src/chainstate/stacks/miner.rs

```diff
@@ -280,8 +280,8 @@ impl StacksBlockBuilder {
 
         // ...and move it (possibly overwriting)
         // TODO: this is atomic but _not_ crash-consistent!
-        fs::rename(&block_pathbuf, &mined_block_pathbuf)
-            .expect(&format!("FATAL: failed to rename {:?} to {:?}", &block_pathbuf, &mined_block_pathbuf));
+        fs::rename(&block_pathbuf, &mined_block_pathbuf);
+            // .expect(&format!("FATAL: failed to rename {:?} to {:?}", &block_pathbuf, &mined_block_pathbuf));
```

Comment:
Why is this commented out?  If the `rename()` fails for any reason, this should panic.

Reply:
Yeah I pushed an additional commit a few minutes after opening the draft :)